renovate.json

```diff
@@ -682,16 +684,884 @@
       ],
       "labels": [
         "Team:Operations",
+        "release_note:skip",
+        "backport:all-open"
+      ],
+      "minimumReleaseAge": "7 days",
+      "enabled": true
+    },
+    {
+      "groupName": "backport",
+      "matchDepNames": [
+        "backport"
+      ],
+      "matchDepPatterns": [],
+      "reviewers": [
+        "team:kibana-operations"
+      ],
+      "matchBaseBranches": [
+        "main"
+      ],
+      "labels": [
+        "Team:Operations",
+        "release_note:skip",
+        "backport:all-open"
+      ],
+      "minimumReleaseAge": "7 days",
+      "enabled": true
+    },
+    {
+      "groupName": "bazel",
```

Comment:
`bazel` will be removed in the near future, so don't think this is needed.

Reply:
I'll mark this as `enabled: false` for now, but I'd prefer to keep it here for my own tracking for the time being.